troubleshoot/proxy/troubleshoot_proxy.go

```diff
@@ -85,10 +85,11 @@ func (t *Troubleshoot) RunAllTests(upstreamEnvoyID, upstreamIP string) (validate
 	}
 
 	// getStats usage example
-	// rejectionStats, err := t.getEnvoyStats("update_rejected")
-	// if err != nil {
-	// 	resultErr = multierror.Append(resultErr, err)
-	// }
+	messages, err = t.troubleshootStats(upstreamEnvoyID)
```

Comment:
Would this only work when an EnvoyID is passed in? What about with vips?

Reply:
I had a conversation with Nitya, and similar to before, we do not have a reliable way of constructing the SNI from envoyID, so I removed it.